_includes/footer.html

```diff
@@ -1,14 +1,52 @@
-<footer>
-	<div class="row text-center">
+<div class="background-dark">
+	<div class="container">
```

Comment:
I think we need more margin at the bottom. It currently seems to be quite tight at the bottom:

![Screenshot from 2024-05-13 13-10-52](https://github.com/precice/precice.github.io/assets/4943683/e538aba9-344a-4d0b-adda-7abd50357744)